libraries/program-error/tests/mod.rs

```diff
@@ -32,6 +33,7 @@ mod tests {
 
     // `#[derive(IntoProgramError)]`
     #[test]
+    #[serial]
```

Comment:
Since this one doesn't use the expected data, it should be fine to run in parallel, right?

Reply:
yeah true, sounds good, done!